framework/wazuh/core/indexer/events.py

```diff
@@ -23,6 +22,7 @@ def __init__(self, client: AsyncOpenSearch):
     async def create(
```

Comment:
Similar to what was mentioned above, we could rename the method.
```suggestion
    async def send(
```